backend/django/core/utils/utils_model.py

```diff
@@ -46,7 +57,10 @@ def cohens_kappa(project):
     labels_seen = set()
     for d in irr_data:
         d_log = IRRLog.objects.filter(data=d, data__project=project)
-        labels = list(set(d_log.values_list('label', flat=True)))
+        # NOTE: Need to figure out why tests fail when this is not there
+        print(list(d_log), len(d_log))
```

Comment:
So this line has been causing me no end of problems. For some reason, when I get rid of this print statement both cohens kappa tests that expect and answer other than 1 consistently fail. I tried replacing the statement with time.sleep(0.5) thinking that it was some sort of race condition, but that didn't fix it.